billing-ui/package.json

```diff
@@ -40,13 +40,13 @@
     "gzip-size": "3.0.0",
     "html-webpack-plugin": "2.22.0",
     "json-loader": "0.5.4",
-    "node-sass": "^3.13.1",
+    "node-sass": "^4.12.0",
```

Comment:
This is a PR for fixing the python version, why are we updating UI dependencies?